From bc59296789c0fcb3d670f1de54283968ebb64d39 Mon Sep 17 00:00:00 2001
From: Minecrell <minecrell@minecrell.net>
Date: Tue, 6 Aug 2019 14:52:42 +0200
Subject: [PATCH 22/78] MSM8916: arm64: configs: Add msm8916_defconfig (v38)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

 v2: CONFIG_DRM_PANEL_BOOYI_OTM1287=m
     CONFIG_LEDS_QCOM_LPG=m
 v3: CONFIG_RMI4_F11=y => CONFIG_RMI4_F12=y
 v4: CONFIG_DUMB_ADC_BATTERY=m
 v5: CONFIG_DRM_MSM=m
     CONFIG_QCOM_A53PLL=y
     CONFIG_QCOM_CLK_APCS_MSM8916=y
     CONFIG_REMOTEPROC=m => CONFIG_REMOTEPROC=y
 v6: CONFIG_QCOM_SOCINFO=y
 v7: CONFIG_DRM_PANEL_SAMSUNG_S6D7AA0_LTL101AT01=m
 v8: CONFIG_TOUCHSCREEN_ATMEL_MXT=m
 v9: CONFIG_BACKLIGHT_PWM=m => CONFIG_BACKLIGHT_PWM=y
     CONFIG_LEDS_CLASS=m -> CONFIG_LEDS_CLASS=y
     CONFIG_LEDS_QCOM_LPG=m => CONFIG_LEDS_QCOM_LPG=y
v10: CONFIG_NR_CPUS=4
     Remove CONFIG_ARM_CPUIDLE=y
     CONFIG_QCOM_PM=y => CONFIG_ARM_QCOM_SPM_CPUIDLE=y
     CONFIG_CPU_FREQ_GOV_POWERSAVE=m => CONFIG_CPU_FREQ_GOV_POWERSAVE=y
     CONFIG_NFC_NXP_NCI=m
     CONFIG_NFC_NXP_NCI_I2C=m
     CONFIG_DRM_PANEL_SAMSUNG_S6E88A0_AMS427AP24=m
     CONFIG_IIO_ST_LSM6DSX=m
v11: CONFIG_RPMSG_CHAR=y
v12: CONFIG_INPUT_PM8XXX_VIBRATOR=m
v13: CONFIG_SND_SOC_QDSP6=m
     CONFIG_QCOM_APR=m
v14: CONFIG_KEYBOARD_TM2_TOUCHKEY=m
v15: CONFIG_SND_SOC_MSM8916_QDSP6=m
v16: CONFIG_RMI4_F11=y
     CONFIG_DRM_PANEL_TIANMA_499=m
     CONFIG_BMA180=m
v17: CONFIG_ARM_QCOM_CPUFREQ_NVMEM=y
     CONFIG_QCOM_CPR=y
     CONFIG_CRYPTO_USER_API_HASH=m
     CONFIG_CRYPTO_USER_API_SKCIPHER=m
     CONFIG_CRYPTO_USER_API_RNG=m
     CONFIG_CRYPTO_USER_API_AEAD=m
v18: CONFIG_DRM_PANEL_SAMSUNG_TC358764_LTL101AL06=m
     CONFIG_IIO_ST_ACCEL_3AXIS=m
v19: CONFIG_SND_SOC_MAX98357A=m
v20: CONFIG_QCOM_FASTRPC=m
     CONFIG_QCOM_RPMPD=y
v21: CONFIG_LEDS_CLASS_FLASH=m
     CONFIG_LEDS_SGM3140=m
v22: CONFIG_DRM_PANEL_YUSHUN_NT35520=m
     CONFIG_V4L2_FLASH_LED_CLASS=m
     CONFIG_LEDS_AW2013=m
     CONFIG_SND_SOC_QDSP6_Q6VOICE=m
     CONFIG_TOUCHSCREEN_HIMAX852X=m
     CONFIG_SND_SOC_TFA9895=m
v23: CONFIG_BATTERY_MAX17042=m
     CONFIG_BMI160_I2C=m
     CONFIG_AK8975=m
     CONFIG_LEDS_GPIO=m
v24: CONFIG_ARM_PSCI_CPUIDLE=y
v25: CONFIG_SND_SOC_AW8738=m (bq-paella)
     CONFIG_DRM_DISPLAY_CONNECTOR=m (DB410c)
     CONFIG_DRM_I2C_ADV7511=m (DB410c)
     CONFIG_DRM_I2C_ADV7511_AUDIO=y (DB410c)
     CONFIG_USB_HSIC_USB3503=y (DB410c)
     CONFIG_LEDS_GPIO=m => CONFIG_LEDS_GPIO=y (DB410c)
v26: CONFIG_DRM_PANEL_BOE_450=m
v27: CONFIG_SCHED_SMT=y
     CONFIG_RMNET=m
     CONFIG_I2C_QCOM_CCI=m
     CONFIG_MEDIA_PLATFORM_SUPPORT=y
     CONFIG_LEDS_TRIGGER_TIMER=y
     CONFIG_INTERCONNECT=y
     CONFIG_INTERCONNECT_QCOM=y
     CONFIG_INTERCONNECT_QCOM_MSM8916=y
v28: CONFIG_QCOM_BAM_DMUX=m
v29: CONFIG_TOUCHSCREEN_EDT_FT5X06=m
     CONFIG_DRM_PANEL_XIAOMI_R61308=m
     CONFIG_SND_SOC_SIMPLE_AMPLIFIER=m
     CONFIG_INV_MPU6050_I2C=m
v30: CONFIG_RFKILL=m => CONFIG_RFKILL=y
     CONFIG_QCOM_SPMI_TEMP_ALARM=m => CONFIG_QCOM_SPMI_TEMP_ALARM=y
     CONFIG_SOUND=m => CONFIG_SOUND=y
     CONFIG_SND=m => CONFIG_SND=y
     CONFIG_SND_SOC=m => CONFIG_SND_SOC=y
     CONFIG_QCOM_RMTFS_MEM=m => CONFIG_QCOM_RMTFS_MEM=y
     CONFIG_IIO=m => CONFIG_IIO=y
     CONFIG_QCOM_SPMI_VADC=m => CONFIG_QCOM_SPMI_VADC=y
v31: CONFIG_EXTCON_USB_DUMMY=1
     CONFIG_DRM_PANEL_OPPO15009JDI_NT35992=m
     CONFIG_BACKLIGHT_LM3630A=m
     CONFIG_TOUCHSCREEN_MELFAS_MIP4=m
     CONFIG_DRM_PANEL_WINGTECH_YASSY_ILI9881=m
     CONFIG_REGULATOR_TPS65132=y
     CONFIG_DRM_PANEL_MOTOROLA_OSPREY_INX=m
     CONFIG_CHARGER_PM8916_LBC=y
     CONFIG_BATTERY_PM8916_BMS_VM=m
     CONFIG_DRM_PANEL_ASUS_Z010D_R69339=m
v32: CONFIG_LEDS_CLASS_MULTICOLOR=y
v33: CONFIG_KXCJK1013=m
v34: CONFIG_PINCTRL_MSM=y
     CONFIG_DRM_PANEL_MSM8916_GENERATED=m
v35: Remove CONFIG_INTERCONNECT{,QCOM,QCOM_MSM8916}=y
     (Not useful at the moment until consumers are added...)
v36: CONFIG_CHARGER_PM8916_LBC=y -> CONFIG_CHARGER_PM8916_LBC=m
v37: CONFIG_STK3310=m
     CONFIG_TOUCHSCREEN_MSG2638=m
     CONFIG_SND_SOC_TFA9895=m => CONFIG_SND_SOC_TFA989X=m
     CONFIG_SMB1360=m
     CONFIG_QCOM_MEMSHARE_QMI_SERVICE=m
v38: CONFIG_TMPFS_POSIX_ACL=y (from upstream)

Co-Authored-By: Nikita Travkin <nikitos.tr@gmail.com>
Co-Authored-By: Jasper Korten <jja2000@gmail.com>
Co-Authored-By: Ruby Iris Juric <ruby@srxl.me>
Co-Authored-By: Gaëtan André <rvlander@gaetanandre.eu>
Co-Authored-By: Jonathan Albrieux <jonathan.albrieux@gmail.com>
Co-Authored-By: Wiktor Strzębała <wiktorek140@tlen.pl>
Co-Authored-By: Devajith V S <devajithvs@gmail.com>
Co-Authored-By: Vincent Knecht <vincent.knecht@mailoo.org>
---
 arch/arm64/configs/msm8916_defconfig | 370 +++++++++++++++++++++++++++
 1 file changed, 370 insertions(+)
 create mode 100644 arch/arm64/configs/msm8916_defconfig

diff --git a/arch/arm64/configs/msm8916_defconfig b/arch/arm64/configs/msm8916_defconfig
new file mode 100644
index 000000000..43a081fce
--- /dev/null
+++ b/arch/arm64/configs/msm8916_defconfig
@@ -0,0 +1,370 @@
+CONFIG_LOCALVERSION="-msm8916"
+CONFIG_SYSVIPC=y
+CONFIG_POSIX_MQUEUE=y
+CONFIG_AUDIT=y
+CONFIG_NO_HZ_IDLE=y
+CONFIG_HIGH_RES_TIMERS=y
+CONFIG_PREEMPT=y
+CONFIG_IRQ_TIME_ACCOUNTING=y
+CONFIG_BSD_PROCESS_ACCT=y
+CONFIG_BSD_PROCESS_ACCT_V3=y
+CONFIG_TASKSTATS=y
+CONFIG_TASK_DELAY_ACCT=y
+CONFIG_TASK_XACCT=y
+CONFIG_TASK_IO_ACCOUNTING=y
+CONFIG_IKCONFIG=y
+CONFIG_IKCONFIG_PROC=y
+CONFIG_MEMCG=y
+CONFIG_BLK_CGROUP=y
+CONFIG_CGROUP_PIDS=y
+CONFIG_CGROUP_HUGETLB=y
+CONFIG_CPUSETS=y
+CONFIG_CGROUP_DEVICE=y
+CONFIG_CGROUP_CPUACCT=y
+CONFIG_CGROUP_PERF=y
+CONFIG_USER_NS=y
+CONFIG_SCHED_AUTOGROUP=y
+CONFIG_BLK_DEV_INITRD=y
+CONFIG_KALLSYMS_ALL=y
+# CONFIG_COMPAT_BRK is not set
+CONFIG_PROFILING=y
+CONFIG_ARCH_QCOM=y
+CONFIG_ARM64_VA_BITS_48=y
+CONFIG_SCHED_MC=y
+CONFIG_SCHED_SMT=y
+CONFIG_NR_CPUS=4
+CONFIG_KEXEC=y
+CONFIG_CRASH_DUMP=y
+CONFIG_COMPAT=y
+CONFIG_RANDOMIZE_BASE=y
+# CONFIG_EFI is not set
+CONFIG_HIBERNATION=y
+CONFIG_WQ_POWER_EFFICIENT_DEFAULT=y
+CONFIG_CPU_IDLE=y
+CONFIG_ARM_PSCI_CPUIDLE=y
+CONFIG_ARM_QCOM_SPM_CPUIDLE=y
+CONFIG_CPU_FREQ=y
+CONFIG_CPU_FREQ_STAT=y
+CONFIG_CPU_FREQ_GOV_POWERSAVE=y
+CONFIG_CPU_FREQ_GOV_USERSPACE=m
+CONFIG_CPU_FREQ_GOV_ONDEMAND=m
+CONFIG_CPU_FREQ_GOV_CONSERVATIVE=m
+CONFIG_CPUFREQ_DT=y
+CONFIG_ARM_QCOM_CPUFREQ_NVMEM=y
+CONFIG_ARM64_CRYPTO=y
+CONFIG_CRYPTO_SHA1_ARM64_CE=y
+CONFIG_CRYPTO_SHA2_ARM64_CE=y
+CONFIG_CRYPTO_SHA512_ARM64_CE=m
+CONFIG_CRYPTO_SHA3_ARM64=m
+CONFIG_CRYPTO_SM3_ARM64_CE=m
+CONFIG_CRYPTO_SM4_ARM64_CE=m
+CONFIG_CRYPTO_GHASH_ARM64_CE=y
+CONFIG_CRYPTO_AES_ARM64_CE_CCM=y
+CONFIG_CRYPTO_AES_ARM64_CE_BLK=y
+CONFIG_CRYPTO_CHACHA20_NEON=m
+CONFIG_CRYPTO_NHPOLY1305_NEON=m
+CONFIG_CRYPTO_AES_ARM64_BS=m
+CONFIG_JUMP_LABEL=y
+CONFIG_MODULES=y
+CONFIG_MODULE_UNLOAD=y
+CONFIG_MODULE_SRCVERSION_ALL=y
+CONFIG_MODULE_COMPRESS=y
+CONFIG_KSM=y
+CONFIG_MEMORY_FAILURE=y
+CONFIG_TRANSPARENT_HUGEPAGE=y
+CONFIG_CMA=y
+CONFIG_NET=y
+CONFIG_PACKET=y
+CONFIG_UNIX=y
+CONFIG_INET=y
+CONFIG_IP_MULTICAST=y
+CONFIG_IP_PNP=y
+CONFIG_IP_PNP_DHCP=y
+CONFIG_IP_PNP_BOOTP=y
+CONFIG_IPV6=m
+CONFIG_NETFILTER=y
+CONFIG_NF_CONNTRACK=m
+CONFIG_NF_CONNTRACK_EVENTS=y
+CONFIG_NETFILTER_XT_TARGET_CHECKSUM=m
+CONFIG_NETFILTER_XT_TARGET_LOG=m
+CONFIG_NETFILTER_XT_MATCH_ADDRTYPE=m
+CONFIG_NETFILTER_XT_MATCH_CONNTRACK=m
+CONFIG_IP_NF_IPTABLES=m
+CONFIG_IP_NF_FILTER=m
+CONFIG_IP_NF_TARGET_REJECT=m
+CONFIG_IP_NF_NAT=m
+CONFIG_IP_NF_TARGET_MASQUERADE=m
+CONFIG_IP_NF_MANGLE=m
+CONFIG_IP6_NF_IPTABLES=m
+CONFIG_IP6_NF_FILTER=m
+CONFIG_IP6_NF_TARGET_REJECT=m
+CONFIG_IP6_NF_MANGLE=m
+CONFIG_IP6_NF_NAT=m
+CONFIG_IP6_NF_TARGET_MASQUERADE=m
+CONFIG_BRIDGE=m
+CONFIG_BRIDGE_VLAN_FILTERING=y
+CONFIG_VLAN_8021Q=m
+CONFIG_VLAN_8021Q_GVRP=y
+CONFIG_VLAN_8021Q_MVRP=y
+CONFIG_QRTR=m
+CONFIG_QRTR_SMD=m
+CONFIG_QRTR_TUN=m
+CONFIG_BPF_JIT=y
+CONFIG_BT=m
+CONFIG_BT_HIDP=m
+CONFIG_BT_LEDS=y
+CONFIG_BT_HCIBTUSB=m
+CONFIG_BT_QCOMSMD=m
+CONFIG_CFG80211=m
+CONFIG_MAC80211=m
+CONFIG_MAC80211_LEDS=y
+CONFIG_RFKILL=y
+CONFIG_NFC=m
+CONFIG_NFC_NCI=m
+CONFIG_NFC_NXP_NCI=m
+CONFIG_NFC_NXP_NCI_I2C=m
+CONFIG_NFC_S3FWRN5_I2C=m
+CONFIG_DEVTMPFS=y
+CONFIG_DEVTMPFS_MOUNT=y
+# CONFIG_QCOM_EBI2 is not set
+CONFIG_OF_OVERLAY=y
+CONFIG_BLK_DEV_LOOP=y
+CONFIG_QCOM_FASTRPC=m
+CONFIG_SCSI=y
+# CONFIG_SCSI_PROC_FS is not set
+CONFIG_BLK_DEV_SD=y
+CONFIG_MD=y
+CONFIG_NETDEVICES=y
+CONFIG_MACVLAN=m
+CONFIG_MACVTAP=m
+CONFIG_VXLAN=m
+CONFIG_TUN=m
+CONFIG_VETH=m
+CONFIG_QCOM_BAM_DMUX=m
+CONFIG_RMNET=m
+CONFIG_WCN36XX=m
+CONFIG_INPUT_EVDEV=y
+# CONFIG_KEYBOARD_ATKBD is not set
+CONFIG_KEYBOARD_GPIO=y
+CONFIG_KEYBOARD_TM2_TOUCHKEY=m
+# CONFIG_MOUSE_PS2 is not set
+CONFIG_INPUT_TOUCHSCREEN=y
+CONFIG_TOUCHSCREEN_ATMEL_MXT=m
+CONFIG_TOUCHSCREEN_MMS114=m
+CONFIG_TOUCHSCREEN_MELFAS_MIP4=m
+CONFIG_TOUCHSCREEN_EDT_FT5X06=m
+CONFIG_TOUCHSCREEN_ZINITIX=m
+CONFIG_TOUCHSCREEN_MSG2638=m
+CONFIG_TOUCHSCREEN_HIMAX852X=m
+CONFIG_INPUT_MISC=y
+CONFIG_INPUT_PM8941_PWRKEY=y
+CONFIG_INPUT_PM8XXX_VIBRATOR=m
+CONFIG_RMI4_CORE=m
+CONFIG_RMI4_I2C=m
+CONFIG_RMI4_F11=y
+CONFIG_RMI4_F12=y
+# CONFIG_SERIO is not set
+CONFIG_LEGACY_PTY_COUNT=16
+CONFIG_SERIAL_MSM=y
+CONFIG_SERIAL_MSM_CONSOLE=y
+CONFIG_SERIAL_DEV_BUS=y
+CONFIG_I2C=y
+CONFIG_I2C_CHARDEV=y
+CONFIG_I2C_MUX=y
+CONFIG_I2C_GPIO=y
+CONFIG_I2C_QCOM_CCI=m
+CONFIG_I2C_QUP=y
+CONFIG_I2C_SLAVE=y
+CONFIG_SPI=y
+CONFIG_SPI_MEM=y
+CONFIG_SPI_BITBANG=m
+CONFIG_SPI_QUP=y
+CONFIG_SPI_SPIDEV=m
+CONFIG_SPMI=y
+CONFIG_PTP_1588_CLOCK=y
+CONFIG_PINCTRL_SINGLE=y
+CONFIG_PINCTRL_MSM=y
+CONFIG_PINCTRL_MSM8916=y
+CONFIG_PINCTRL_QCOM_SPMI_PMIC=y
+CONFIG_POWER_RESET_MSM=y
+CONFIG_POWER_RESET_QCOM_PON=y
+CONFIG_BATTERY_MAX17042=m
+CONFIG_SMB1360=m
+CONFIG_BATTERY_PM8916_BMS_VM=m
+CONFIG_CHARGER_PM8916_LBC=m
+CONFIG_BATTERY_RT5033=m
+CONFIG_THERMAL=y
+CONFIG_CPU_THERMAL=y
+CONFIG_THERMAL_EMULATION=y
+CONFIG_QCOM_TSENS=y
+CONFIG_QCOM_SPMI_TEMP_ALARM=y
+CONFIG_WATCHDOG=y
+CONFIG_PM8916_WATCHDOG=y
+CONFIG_MFD_SPMI_PMIC=y
+CONFIG_MFD_RT5033=m
+CONFIG_REGULATOR_FIXED_VOLTAGE=y
+CONFIG_REGULATOR_QCOM_SMD_RPM=y
+CONFIG_REGULATOR_QCOM_SPMI=y
+CONFIG_REGULATOR_RT5033=m
+CONFIG_REGULATOR_TPS65132=y
+CONFIG_MEDIA_SUPPORT=m
+CONFIG_MEDIA_CAMERA_SUPPORT=y
+CONFIG_MEDIA_PLATFORM_SUPPORT=y
+CONFIG_V4L2_FLASH_LED_CLASS=m
+CONFIG_V4L_PLATFORM_DRIVERS=y
+CONFIG_VIDEO_QCOM_CAMSS=m
+CONFIG_V4L_MEM2MEM_DRIVERS=y
+CONFIG_VIDEO_QCOM_VENUS=m
+CONFIG_DRM=m
+CONFIG_DRM_MSM=m
+CONFIG_DRM_PANEL_SIMPLE=m
+CONFIG_DRM_PANEL_SAMSUNG_S6E88A0_AMS452EF01=m
+CONFIG_DRM_PANEL_MSM8916_GENERATED=m
+CONFIG_DRM_DISPLAY_CONNECTOR=m
+CONFIG_DRM_I2C_ADV7511=m
+CONFIG_DRM_I2C_ADV7511_AUDIO=y
+CONFIG_DRM_LEGACY=y
+CONFIG_FB=y
+CONFIG_FB_MODE_HELPERS=y
+CONFIG_FB_SIMPLE=y
+CONFIG_BACKLIGHT_CLASS_DEVICE=y
+CONFIG_BACKLIGHT_PWM=y
+CONFIG_BACKLIGHT_LM3630A=m
+CONFIG_FRAMEBUFFER_CONSOLE_ROTATION=y
+CONFIG_SOUND=y
+CONFIG_SND=y
+CONFIG_SND_DYNAMIC_MINORS=y
+CONFIG_SND_SOC=y
+CONFIG_SND_SOC_QCOM=m
+CONFIG_SND_SOC_APQ8016_SBC=m
+CONFIG_SND_SOC_QDSP6_Q6VOICE=m
+CONFIG_SND_SOC_MSM8916_QDSP6=m
+CONFIG_SND_SOC_AW8738=m
+CONFIG_SND_SOC_MAX98357A=m
+CONFIG_SND_SOC_MSM8916_WCD_ANALOG=m
+CONFIG_SND_SOC_MSM8916_WCD_DIGITAL=m
+CONFIG_SND_SOC_SIMPLE_AMPLIFIER=m
+CONFIG_SND_SOC_TFA989X=m
+CONFIG_USB_LED_TRIG=y
+CONFIG_USB=y
+CONFIG_USB_OTG=y
+CONFIG_USB_EHCI_HCD=y
+CONFIG_USB_STORAGE=y
+CONFIG_USB_CHIPIDEA=y
+CONFIG_USB_CHIPIDEA_UDC=y
+CONFIG_USB_CHIPIDEA_HOST=y
+CONFIG_USB_HSIC_USB3503=y
+CONFIG_USB_GADGET=y
+CONFIG_MMC=y
+CONFIG_MMC_BLOCK_MINORS=32
+CONFIG_MMC_SDHCI=y
+CONFIG_MMC_SDHCI_PLTFM=y
+CONFIG_MMC_SDHCI_MSM=y
+CONFIG_NEW_LEDS=y
+CONFIG_LEDS_CLASS=y
+CONFIG_LEDS_CLASS_FLASH=m
+CONFIG_LEDS_CLASS_MULTICOLOR=y
+CONFIG_LEDS_AW2013=m
+CONFIG_LEDS_GPIO=y
+CONFIG_LEDS_QCOM_LPG=y
+CONFIG_LEDS_SGM3140=m
+CONFIG_LEDS_TRIGGER_TIMER=y
+CONFIG_LEDS_TRIGGER_HEARTBEAT=y
+CONFIG_LEDS_TRIGGER_CPU=y
+CONFIG_LEDS_TRIGGER_DEFAULT_ON=y
+CONFIG_LEDS_TRIGGER_PANIC=y
+CONFIG_RTC_CLASS=y
+CONFIG_RTC_DRV_PM8XXX=y
+CONFIG_DMADEVICES=y
+CONFIG_QCOM_BAM_DMA=y
+CONFIG_COMMON_CLK_QCOM=y
+CONFIG_QCOM_A53PLL=y
+CONFIG_QCOM_CLK_APCS_MSM8916=y
+CONFIG_QCOM_CLK_SMD_RPM=y
+CONFIG_MSM_GCC_8916=y
+CONFIG_HWSPINLOCK=y
+CONFIG_HWSPINLOCK_QCOM=y
+CONFIG_MAILBOX=y
+CONFIG_QCOM_APCS_IPC=y
+CONFIG_QCOM_IOMMU=y
+CONFIG_REMOTEPROC=y
+CONFIG_QCOM_Q6V5_MSS=m
+CONFIG_QCOM_SYSMON=m
+CONFIG_QCOM_WCNSS_PIL=m
+CONFIG_RPMSG_CHAR=y
+CONFIG_RPMSG_QCOM_GLINK_SMEM=m
+CONFIG_RPMSG_QCOM_SMD=y
+CONFIG_QCOM_CPR=y
+CONFIG_QCOM_RMTFS_MEM=y
+CONFIG_QCOM_RPMPD=y
+CONFIG_QCOM_SMEM=y
+CONFIG_QCOM_SMD_RPM=y
+CONFIG_QCOM_SMP2P=y
+CONFIG_QCOM_SMSM=y
+CONFIG_QCOM_SOCINFO=y
+CONFIG_QCOM_WCNSS_CTRL=m
+CONFIG_QCOM_APR=m
+CONFIG_QCOM_MEMSHARE_QMI_SERVICE=m
+CONFIG_PM_DEVFREQ=y
+CONFIG_DEVFREQ_GOV_SIMPLE_ONDEMAND=y
+CONFIG_EXTCON_SM5502=y
+CONFIG_EXTCON_USB_DUMMY=y
+CONFIG_EXTCON_USB_GPIO=y
+CONFIG_MEMORY=y
+CONFIG_IIO=y
+CONFIG_BMA180=m
+CONFIG_BMC150_ACCEL=m
+CONFIG_IIO_ST_ACCEL_3AXIS=m
+CONFIG_KXCJK1013=m
+CONFIG_QCOM_SPMI_VADC=y
+CONFIG_BMG160=m
+CONFIG_BMI160_I2C=m
+CONFIG_INV_MPU6050_I2C=m
+CONFIG_IIO_ST_LSM6DSX=m
+CONFIG_LTR501=m
+CONFIG_STK3310=m
+CONFIG_AK8975=m
+CONFIG_BMC150_MAGN_I2C=m
+CONFIG_PWM=y
+CONFIG_PHY_QCOM_USB_HS=y
+CONFIG_QCOM_QFPROM=y
+CONFIG_EXT4_FS=y
+CONFIG_EXT4_FS_POSIX_ACL=y
+CONFIG_FANOTIFY=y
+CONFIG_FANOTIFY_ACCESS_PERMISSIONS=y
+CONFIG_QUOTA=y
+CONFIG_AUTOFS4_FS=y
+CONFIG_FUSE_FS=m
+CONFIG_CUSE=m
+CONFIG_OVERLAY_FS=m
+CONFIG_VFAT_FS=y
+CONFIG_TMPFS_POSIX_ACL=y
+CONFIG_HUGETLBFS=y
+CONFIG_CONFIGFS_FS=y
+CONFIG_PSTORE=y
+CONFIG_PSTORE_ZSTD_COMPRESS=y
+CONFIG_PSTORE_ZSTD_COMPRESS_DEFAULT=y
+CONFIG_NLS_CODEPAGE_437=y
+CONFIG_NLS_ISO8859_1=y
+CONFIG_SECURITY=y
+CONFIG_SECURITYFS=y
+CONFIG_CRYPTO_NULL=y
+CONFIG_CRYPTO_SHA512=m
+CONFIG_CRYPTO_ANSI_CPRNG=y
+CONFIG_CRYPTO_DRBG_MENU=y
+CONFIG_CRYPTO_USER_API_HASH=m
+CONFIG_CRYPTO_USER_API_SKCIPHER=m
+CONFIG_CRYPTO_USER_API_RNG=m
+CONFIG_CRYPTO_USER_API_AEAD=m
+CONFIG_CRYPTO_DEV_QCOM_RNG=y
+CONFIG_DMA_CMA=y
+CONFIG_CMA_SIZE_MBYTES=32
+CONFIG_PRINTK_TIME=y
+CONFIG_DYNAMIC_DEBUG=y
+CONFIG_DEBUG_INFO=y
+CONFIG_MAGIC_SYSRQ=y
+CONFIG_DEBUG_FS=y
+CONFIG_DEBUG_KERNEL=y
+# CONFIG_SCHED_DEBUG is not set
+# CONFIG_DEBUG_PREEMPT is not set
+# CONFIG_FTRACE is not set
-- 
2.31.1

